AirCasting/AuthViews/SignInView.swift

```diff
@@ -170,6 +183,56 @@ private extension SignInView {
                          dismissButton: .default(Text("Ok")))
         }
     }
+    
+    struct ForgotPasswordView: View {
+        @Environment(\.presentationMode) var presentationMode
+        @Binding var presentedAsModal: Bool
+        @State private var showingAlert = false
+        @State private var email = ""
+        @State private var popUpMessage = ""
+        var body: some View {
+            VStack(alignment: .leading, spacing: 30) {
+                title
+                createTextfield(placeholder: "email or username", binding: $email)
+                description
+                sendButton
+            }.padding()
+        }
+        
+        private var title: some View {
+            Text("Forgot Password")
+                .font(Font.moderate(size: 32, weight: .bold))
+                .foregroundColor(.accentColor)
+        }
+        
+        private var description: some View {
+            Text("You will get en email with details after 'send new' button pressed")
+                .font(Font.muli(size: 16))
+                .foregroundColor(.aircastingGray)
+        }
+        
+        var sendButton: some View {
+            Button("Send new") {
+                APICommunication().forgotPassword(login: email) { value in
```

Comment:
I feel like this should be injected into here. Also views calling services directly is not a greatest idea, consider changing architecture to a layered one (add a viewmodel or/and a controller that would do it instead). We can schedule a call and pair-program it if you want.